apps/src/calc/calc.js

```diff
@@ -202,6 +202,27 @@ Calc.init = function(config) {
     // base's studioApp.resetButtonClick will be called first
     var resetButton = document.getElementById('resetButton');
     dom.addClickTouchEvent(resetButton, Calc.resetButtonClick);
+
+    if (Blockly.contractEditor) {
+      Blockly.contractEditor.registerTestHandlers(function (exampleBlock) {
+        try {
+          var entireSet = new EquationSet(Blockly.mainBlockSpace.getTopBlocks());
+
+          var actualBlock = exampleBlock.getInputTargetBlock("ACTUAL");
+          var actualEquation = EquationSet.__testonly__.getEquationFromBlock(actualBlock);
```

Comment:
This doesn't look like test code. Probably just means we need to expose getEquationFromBlock somewhere that isn't listed as testonly.